File: php/lab2_php/print_even_and_odd_sums.php
<?php

/**
 * Reads n numbers and adds even and odd ones separately.
 *
 * Prints even and odd results.
 */
function printEvenAndOddSums()
{
    $n = readline("How many numbers? ");

    $oddSum = 0;
    $evenSum = 0;

    for ($i = 0; $i < $n; $i++) {
        $nr = readline("Enter a number: ");
        $nr = (int) $nr;

        if ($nr % 2 == 0) {
            $evenSum += $nr;
            continue;
        }

        $oddSum += $nr;
    }

    echo "Odd sum is: {$oddSum}";
    echo "Even sum is: {$evenSum}";
}

printEvenAndOddSums();

File: cpp/lab2/print_even_and_odd_sums.cpp
#include <cstdio>
#include <iostream>
#include "lab2.h"

/**
 * Reads n numbers and adds even and odd ones separately.
 *
 * Prints even and odd results.
 */
void printEvenAndOddSums()
{
    std::cout << "How many numbers should be added? \n";

    int n;
    std::cin >> n;

    int oddSum = 0;
    int evenSum = 0;
    int current = 0;
    while (n > 0) {
        std::cout << "Enter a number: \n";
        std::cin >> current;
        n--;

        if ((current % 2) == 0) {
            evenSum += current;
            continue;
        }

        oddSum += current;
    }

    std::cout << "The odd sum is " << oddSum << " and the even sum is " << evenSum;
}